src/main/java/com/minerarcana/floralchemy/FloraObjectHolder.java

```diff
@@ -0,0 +1,9 @@
+package com.minerarcana.floralchemy;
+
+import net.minecraft.block.Block;
+import net.minecraftforge.fml.common.registry.GameRegistry.ObjectHolder;
+
+@ObjectHolder(value = Floralchemy.MOD_ID)
+public class FloraObjectHolder {
+    public static final Block LEAKY_CAULDRON = null;
```

Comment:
You mean need to add another object holder as the name doesn't match exactly any more

Reply:
Ah yes, casing matters. Why do you want to follow the Java naming convention being that this is an edgecase?

Reply:
You can add an objectholder annotation on the field as well, you know that right.

Reply:
You only need the object name, not the modid if you have the class annotation